test/controllers/graphql_controller_test.rb

```diff
@@ -378,7 +378,7 @@ def setup
         create_tag annotated: pm, annotator: u
         create_dynamic_annotation annotated: pm, annotator: u, annotation_type: 'metadata'
       end
-      query = "query GetById { project_media(ids: \"#{pm.id},#{p.id}\") { last_status, domain, pusher_channel, account { url }, dbid, annotations_count(annotation_type: \"comment\"), user { name }, tags(first: 1) { edges { node { tag } } }, project { title }, log(first: 1000) { edges { node { event_type, object_after, updated_at, created_at, meta, object_changes_json, user { name }, annotation { id, created_at, updated_at }, task { id }, tag { id } } } } } }"
```

Comment:
You didn't need to delete `user { name }` from this test, right?

Reply:
Yes indeed, fixed!